README.md

```diff
@@ -345,6 +346,48 @@ To capture data from EventSources running in the same process as EventFlow, the
 
 (*) Either providerName, or providerGuid must be specified. When both are specified, provider GUID takes precedence.
 
+#### Application Insights input
+
+*Nuget package:* [**Microsoft.Diagnostics.EventFlow.Inputs.EventFlowTelementryProcessor**](https://www.nuget.org/packages/Microsoft.Diagnostics.EventFlow.Inputs.EventFlowTelemetryProcessor/)
+
+Application Insights input is designed for the following scenario:
+1. You have an application that uses Application Insights for monitoring and diagnostics.
+2. You want to send a portion of your Application Insights telemetry to some destination other than Application Insights (e.g. Azure EventHub or Elasticsearch; the assumption is there is an EventFlow output for where the data needs to go).
```

Comment:
would be great to provide an explanaition. For instance - if this processor is before sampling processor - Applicaito nInsights will see sampled data when Elastic Search will store everything